hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -2367,4 +2373,170 @@ final Stream<DynamicTest> createHollowAccountWithNftTransferAndCompleteIt() {
+        final AtomicReference<ByteString> hollowAccountAlias = new AtomicReference<>();
+        final var transferHBARSToHollowAccountTxn = "transferHBARSToHollowAccountTxn";
+        return propertyPreservingHapiSpec("createHollowAccountWithHBARSTransferAndCompleteIt")
+                .preserving("entities.unlimitedAutoAssociationsEnabled")
+                .given(
+                        overriding("entities.unlimitedAutoAssociationsEnabled", TRUE_VALUE),
+                        newKeyNamed(hollowAccountKey).shape(SECP_256K1_SHAPE),
+                        cryptoCreate(TREASURY).balance(10_000 * ONE_MILLION_HBARS),
+                        withOpContext((spec, opLog) -> {
+                            final var registry = spec.registry();
+                            final var treasuryAccountId = registry.getAccountID(TREASURY);
+                            treasuryAlias.set(ByteString.copyFrom(asSolidityAddress(treasuryAccountId)));
+                            // Save the alias for the hollow account
+                            final var ecdsaKey = spec.registry()
+                                    .getKey(hollowAccountKey)
+                                    .getECDSASecp256K1()
+                                    .toByteArray();
+                            final var evmAddressBytes = ByteString.copyFrom(recoverAddressFromPubKey(ecdsaKey));
+                            hollowAccountAlias.set(evmAddressBytes);
+                        }))
+                .when(withOpContext((spec, opLog) -> {
+                    // Create a hollow account
+                    var hollowCreate = cryptoCreate(hollowAccountKey)
+                            .key(hollowAccountKey)
+                            .maxAutomaticTokenAssociations(0)
+                            .alias(hollowAccountAlias.get())
+                            .via(transferHBARSToHollowAccountTxn);
+
+                    final HapiGetTxnRecord hapiGetTxnRecord = getTxnRecord(transferHBARSToHollowAccountTxn)
+                            .andAllChildRecords()
+                            .assertingNothingAboutHashes();
+                    allRunFor(spec, hollowCreate, hapiGetTxnRecord);
+                    if (!hapiGetTxnRecord.getChildRecords().isEmpty()) {
+                        final var newAccountID = hapiGetTxnRecord
+                                .getFirstNonStakingChildRecord()
+                                .getReceipt()
+                                .getAccountID();
+                        spec.registry().saveAccountId(hollowAccountKey, newAccountID);
+                    }
+                    // Verify maxAutomaticAssociations is set to 0
+                    var getInfo = getAccountInfo(hollowAccountKey)
+                            .hasAlreadyUsedAutomaticAssociations(0)
+                            .has(accountWith().key(hollowAccountKey))
+                            .hasMaxAutomaticAssociations(0)
```

Comment:
If we wanna have hollow account with maxAutoAssociation = 0, initially we need to be working with maxAutoAssociations flag == false. Then after the first delete we need to find out if we can switch the flag to true before executing the next hollow account creation. After it the hollow account created will have maxAutoAssociations = 1.

Reply:
Where is the flag for my case? I see a flag only in CryptoUpdateSuite.

Reply:
This is the flag -> overriding("entities.unlimitedAutoAssociationsEnabled", TRUE_VALUE),

Reply:
Thank you!